apps/opencs/view/world/scriptsubview.cpp

```diff
@@ -198,26 +198,38 @@ void CSVWorld::ScriptSubView::useHint (const std::string& hint)
     if (hint.empty())
         return;
 
-    if (hint[0]=='l')
-    {
-        std::istringstream stream (hint.c_str()+1);
-
-        char ignore;
-        int line;
-        int column;
-
-        if (stream >> ignore >> line >> column)
+    size_t line = 0, column = 0;
+    std::istringstream stream (hint.c_str()+2);
```

Comment:
Even internally we should shield ourself against undefined behaviour in case of errors where we can. Defensive coding and such. But once we get past 1.0 and have editor plugins the hint system will be exposed to plugins. Having a broken plugin make OpenMW-CS crash with a segfault would be bad.

> But I can't understand how this check works: "if (stream >> ignore >> line >> column)". >> operator returns istream&, that somehow casts to bool, but according to this, bool operator was introduced in c++11.

Nope. That worked since c++98 at least. Probably even pre-standard C++. Implicit cast to const void * IIRC.